learn/os_setup/choose_compiler.md

```diff
@@ -6,8 +6,6 @@ permalink: /learn/os_setup/choose_compiler
 
 A comprehensive list of available compilers is provided [here]({{site.baseurl}}/compilers). In this guide, we will focus only on those that are free to install and use.
 
-Of those listed at the link above, open source and free are GFortran (with OpenCoarrays), Flang, and LFortran. Finally, NVFortran and IFort* are commercial with free community editions.
+Of those listed at the link above, open source and free are GFortran (with OpenCoarrays), Flang, and LFortran. NVIDIA and INTEL offer their compilers for free and as of the latest update of this tutorial, NVFortran (NVIDIA) is available only for Linux systems, while Intel oneAPI is available for Linux, Windows and macOS systems. Both are well documented and readers can find detailed information about installing and using them on their websites respectively. 
```

Comment:
I think INTEL should be spelled as Intel.

Reply:
Sorry, @certik I agree, but was too busy at work and missed the opportunity to correct things in time.

Reply:
No worries @smeskos. Do you have time to submit a PR fixing this?